src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Climber extends SubsystemBase {
+    CANSparkMax spark1 = new CANSparkMax(Constants.CLIMBER_MOTOR_ID, MotorType.kBrushless);
```

Comment:
This is specific to a particular robot. Can we make it generic somehow?

If not, maybe we should name these classes that are year-specific with the year in the name, something like `Climber2020`.

Reply:
I was thinking about doing that